blockchain/msgs_test.go

```diff
@@ -82,7 +82,7 @@ func TestBcStatusResponseMessageValidateBasic(t *testing.T) {
 func TestBlockchainMessageVectors(t *testing.T) {
 	chainLock := types.NewMockChainLock(1)
 
-	block := types.MakeBlock(int64(3),chainLock.CoreBlockHeight,&chainLock,[]types.Tx{types.Tx("Hello World")}, nil, nil)
+	block := types.MakeBlock(int64(3),chainLock.BlockHeight,&chainLock,[]types.Tx{types.Tx("Hello World")}, nil, nil)
```

Comment:
This is very confusing. As this could mean the tendermint BlockHeight at which the chain lock was introduced to platform.

Reply:
Renamed everywhere to coreChainLock. It kinda lools strange `coreChainLock.CoreBlockHeight`. It's already obvious that it's core height by context. Usually, we don't name `block.blockId`, `block.blockHeight` coz it
s obvious by instance/struct name.

Reply:
actually blocks do have a blockId

Reply:
Hehe.. It was a bad example. You caught me.